<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GxsCircleItem</class>
 <widget class="QWidget" name="GxsCircleItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>104</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="GxsCircleItemGLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>215</red>
           <green>215</green>
           <blue>215</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QLabel" name="logoLabel">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/icons/svg/circles.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="forLabel">
          <property name="text">
           <string>for identity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconLabel">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GxsIdLabel" name="gxsIdLabel">
          <property name="text">
           <string notr="true">name</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>358</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="acceptButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/accepted16.png</normaloff>:/images/accepted16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="revokeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Revoke</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="expandButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Details</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/informations_24x24.png</normaloff>:/images/informations_24x24.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="tollbarHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>285</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Remove Item</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/close_normal.png</normaloff>:/images/close_normal.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <layout class="QHBoxLayout" name="nameHLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string notr="true">Circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string notr="true">name</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="nameHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GxsIdLabel</class>
   <extends>QLabel</extends>
   <header>gui/gxs/GxsIdLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
